GeometryCore/src/Polygon.cpp

```diff
@@ -177,9 +177,31 @@ namespace geom_utils
     // For non-convex polygons it should return just empty vector.
     std::vector<Triangle2D> Polygon::triangulate() const
     {
-        return {}; 
+        std::vector <Triangle2D> vectorForTriangulate;
+
+       // FPoint2D centerPoly = centroid();
```

Comment:
Never commit commented out code lines (at least, never let them get into pull requests)
So, please clean all of those